backend/test/test_utils/pipeline_utils.go

```diff
@@ -58,6 +59,28 @@ func ListPipelines(client *api_server.PipelineClient, namespace *string) []*pipe
 	return pipelines
 }
 
+// UploadPipeline - Upload a pipeline
+func UploadPipeline(pipelineUploadClient api_server.PipelineUploadInterface, pipelineFilePath string, pipelineName *string, pipelineDisplayName *string) (*model.V2beta1Pipeline, error) {
+	uploadParams := upload_params.NewUploadPipelineParams()
+	uploadParams.SetName(pipelineName)
+	if pipelineDisplayName != nil {
+		uploadParams.SetDisplayName(pipelineDisplayName)
+	}
+	logger.Log("Creating temp pipeline file with overridden SDK Version")
+	overriddenPipelineFileWithSDKVersion := ReplaceSDKInPipelineSpec(pipelineFilePath, false, nil)
+	tempPipelineFile := CreateTempFile(overriddenPipelineFileWithSDKVersion)
+	defer func() {
+		// Ensure the temporary file is removed when the function exits
+		if err := os.Remove(tempPipelineFile.Name()); err != nil {
+			logger.Log("Error removing temporary file: %s", err)
+		} else {
+			logger.Log("Temporary file removed: %s", tempPipelineFile.Name())
```

Comment:
This is going to be too verbose since every single test will log this. We can just remove the successful case.